lib/Partitioner/Partitioner.cpp

```diff
@@ -1177,6 +1188,28 @@ llvm::Error Partitioner::loadBalancedPartitioning(Function *F,
   return llvm::Error::success();
 }
 
+llvm::Error Partitioner::QuantizationProfilingPartition(
+    CompilationContext &cctx, Function *F, std::vector<Backend *> backends) {
+  FunctionToBackendNameMap funcToBackend;
+  funcToBackend = backendBasedPartition(F_, backends, cctx);
+  module_->eraseFunction(F_);
+  auto backend = createBackend("CPU");
+  // Leave the partitioner early due to quantization profiling mode.
```

Comment:
Is it still a correct comment? Where is the early exit? Is it this `RETURN_IF_ERR ` below?

Reply:
Will move this comment to end of this function for better understanding.